File: DailyFlash Solutions/DailyFlash Solutions/61-DailyFlash_Solutions/31_Mar_Solution_Three/C/Program0.c
#include<stdio.h>

double pow(double,double);

int calculateFactorial(int num){

	int fact =1;
	while(num>0){
		fact *=num;
		num -= 1;
	}

	return fact;
}



void main(){

	int a;
	double res=0;

	printf("Enter the limit : ");
	scanf("%d",&a);


	for(int i=1; i<=a; i++)
		res+=(double)(i)/(calculateFactorial(i));
		

	printf("Result of series is %.2lf\n",res);

}

File: DailyFlash Solutions/DailyFlash Solutions/61-DailyFlash_Solutions/31_Mar_Solution_Three/Java/Program0.java
import java.io.*;

class Program0{


	static int calculateFactorial(int num){

		int fact =1;

		while(num>0){
			fact *=num;
			num -= 1;
		}

		return fact;
	}



	public static void main(String[] args)throws IOException{

		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		int a;
		float res=0;

		while(true){

			try{

				System.out.println("Enter the length of Series : ");
				a = Integer.parseInt(br.readLine());
				break;

			}catch(NumberFormatException nfe){

				System.out.println("NFE : Enter valid number !!!");
			}
		}



		for(int i=1; i<=a; i++)
			res+= (float)i/calculateFactorial(i);



		System.out.printf("Addition of series is : %.2f\n",res);


	}

}
